app/assets/javascripts/components/calendars/appointments.es6

```diff
@@ -0,0 +1,40 @@
+module.exports = {
```

Comment:
As we'll have multiple calendars for appointments, can we rename this to something like `guider_schedule` or similar?